src/utils.rs

```diff
@@ -141,12 +141,34 @@ impl Env {
     }
 }
 
+fn find_config_file() -> Option<PathBuf> {
+    let xdg_config_path = dirs::config_local_dir().unwrap().join("discrakt");
+    let mut exe_path = env::current_exe().unwrap();
+    exe_path.pop();
+
+    let locations = vec![xdg_config_path, exe_path];
```

Comment:
```suggestion
    let config_path = dirs::config_local_dir().unwrap().join("discrakt");
    let mut exe_path = env::current_exe().unwrap();
    exe_path.pop();

    let locations = vec![config_path, exe_path];
```
If `dirs::config_local_dir()` works for all OSs, probably the best naming is simply `config_path`?